asset/testdata/asset_tlv_encoding.json

```diff
@@ -0,0 +1,177 @@
0d77b08ffa57b1dc712aabe44e09ea285a0ef50000000000000000b9c608c75bc18fc98f161622c877a422bffeebcef04ed9905767977cd1c7f2cc0000000000000000f0bcb7d86dc0d48fc48fcd0d11da647ecd1c3549d8cbde20e5bac5da18b913630000000000000000fe4202d6777191febf25953a1ea8c1f320dc7e504ab47a4bb294d6f0ed7db31a0000000000000000c1d8a912dc87d709492ebc43cdb455a2d8a97946c8621c634008d102e734d32a0000000000000000d604ca0d3eab232f4e2940163de8a251f0e18c8be906f93a3bc2fd1aba7b7b53000000000000000041cf4ae56ed981cd9bdc9683b9431e77c6158d4761d174e8a59b59063795b36b00000000000000003904b025d0a4e3baa1e3d465e546d3522663e30b8771f1bfe6798a0c8993a3e900000000000000003f60555aaaf52164e662c130fb4cba80c76f1dd6ae874c7ab042384affe5f16800000000000000007a659e4406a0198dbfae13a2b6475df88e617c34e31571defb6088e3aaf493cb0000000000000000834563ff1274aa1dab643b8cf6743dd75d24e9405c658e28eaf6b4bf860c9c3100000000000000007c9ec4048ca793473058dbd8d2779b295b9be7842dc0afa839e60797f815d22f00000000000000006fc0ab9322188733cf05fa7fbdf2e84697aac090695d94a38e4e91a334e6902e0000000000000000b570fc8f3c92725895a743534485d1fb8771735631985f7464f014d2159848e9000000000000000072d00b5cadb331bdd80e9d683891a6b272b4eb9e26b3e2502fa9576005f3080f000000000000000007780bc3ae3733959e06446eeaec1123de1b9ad71d5a14eddfc4652188e4eb1100000000000000006db868361129285aeb542a3f8ed683895e75965dda29b2d0dcff2e7e63f3c9b10000000000000000df532b345eccd565ccd6e7730422d7993a36ca9a3f7036378e96a57c38383d7a0000000000000000c0db25d52879aef53e310593e346fc4f0ccd9e60847fdf55b995e955455b345f000000000000000019ddf72883b507916d9e63f679dbd4b171058c992301a676165609744bb5206d0000000000000000cecc8126135333b32a4fa1eb35c96370b95c55f216abe81bc1b742bbb0292a6c0000000000000000f41887d1f352a2bc614c0db152932619fe892cb9c19eecf12c5195037a365042000000000000000077bb1e77db06fdade7f031b60c4135b77b73db113a7b1557fbdd5dc8812fa9e00000000000000000984935e54049d8e034222d45ca161aac3d6a2c8f555b596afe99c1f386fd345300000000000000008ead572e77bf1522e570e306096945faa70c27cf7fc92e4e412872865ebcd20d000000000000000042af08a53490158553d5138cfe380acfb49d6e1b4d3b223f1c9962bf9e0767260000000000000000a7d596226ff12cc379a1c733f57f8adc54e6f58f572b1da849122381651b02fa0000000000000000cb710c2bc64511ee93f9c6c09c530c9eead588ffe3a3bf81dfcaead04abb9627000000000000000028c5d45761cf38ed4fb2d4820bc57b9e591a04e9623b1f46da49f3ffa9dbf506000000000000000038f542347e7b8ee5d194890efc220bdf3f49d967c8a3e69d7c113aa0a95d290d0000000000000000b024dbaa5aa401f43071f273fdc98d13381628fe6b5cefd482ca8d64275cf29b00000000000000008f945cfd43b0996b000314bc186816cae2e6698c11ca3749b2d7a00718fb8c3f000000000000000017a3d2b53dfb08a1ab386e6c6f846c78d1aee8c4521e558c01dafecfcda757f0000000000000000013fefdf542341f1f92be3d67058cc4e7eab1303c48ee7fb8873c9dd56c30eedc0000000000000000dd8023b710a813409dd35d7ffab8c71a7625eb15b5bdb981862a5d270b1622a30000000000000000a9ed7261d36f1df934db5cfc81e1ea6e7c7117203dc4b88ccf336e6385153ec100000000000000005a61e238f07e3a8114e39670c1e5ff430913d5793028258cf8a49282efee441100000000000000000000000000000000000000000000000001000000000000000000000000000000",
```

Comment:
Proof seems kiiinda large here? Would expect smaller with just a simple asset and a compressed inclusion proof with a single leaf.